.github/workflows/trigger-gitlab-ctests.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           # created via Settings > CI/CD > Pipeline triggers in your GitLab project
           curl --verbose --request POST \
             --form "token=${{ secrets.GITLAB_TRIGGER_TOKEN }}" \
-            --form "ref=ci_ctests_finalize" \
+            --form "ref=develop" \
             --form "variables[PR_NUMBER]=${{ env.PR_NUM }}" \
             --form "variables[GW_REPO_URL]=${{ vars.GW_REPO_URL }}" \
```

Comment:
Just curious where `GW_REPO_URL` gets set.  Is this the user's repo or the authoritative?

Reply:
hmm ... ❓